### ISSUE TITLE
Redirect

### ISSUE THREAD

**Issue by Genius398** [Educational-Websites/educational-websites.github.io#5]
we should add a home button on every project to redirect people here

**Comment by e-coders** [comment]
Okay!

**Comment by e-coders** [comment]
Will added Fastly!!!!

**Comment by Genius398** [comment]
> Will added Fastly!!!!

AND PLS MAKE PULL REQUESTS NIT MAIN MERGE

**Comment by Super45coder** [comment]
Sure.
________________________________

**Comment by Genius398** [comment]
anyone working on this? I'll help tomorrow

**Comment by somePythonProgrammer** [comment]
Is this done? I think we should add a 'star us' button on our websites.

**Comment by Genius398** [comment]
no, a repository button

**Comment by e-coders** [comment]
I'm busy!! I can talk.

**Comment by somePythonProgrammer** [comment]
> no, a repository button

Ok. we can add a repository button which also shows our stars, I've seen it somewhere.

**Comment by somePythonProgrammer** [comment]
> I'm busy!! I can talk.

Also, I'm confused. How can you talk when you're busy?

**Comment by somePythonProgrammer** [comment]
This might be useful: [LINK](https://buttons.github.io)

**Comment by Super45coder** [comment]
Yeah. it is. I think we should open up sponsors. I can't do it here, so I will make @Genius398 an owner so he can do it in Journ12 Journals. You don't have to agree.
________________________________

**Comment by Super45coder** [comment]
Actually, never mind.
________________________________

**Comment by Genius398** [comment]
It's done but not fully

**Comment by Super45coder** [comment]
Kay.
________________________________

**Comment by Genius398** [comment]
Not doing this now
